frontend/src/views/auth/Login.vue

```diff
@@ -117,6 +118,15 @@ export default {
       }
     })
   },
+  mounted () {
+    const languages = navigator.languages || [navigator.language]
+    for (const language of languages) {
+      if (VueI18n.availableLocales.includes(language)) {
```

Comment:
Müsste man hier nicht eher `this.$i18n.availableLocales` abfragen?

Reply:
stimmt.
Zudem habe ich da jetzt 'browserPreferredLocale' implementiert, so dass man das nicht jedes Mal selber machen muss.